scripts/agent.yml

```diff
@@ -159,6 +158,10 @@ tasks:
         params:
           key: "test"
           destination: "test_num"
+      - command: downstream_expansions.set
+        params:
+          file: ../testdata/downstream_expansions.yml
```

Comment:
Can we put the downstream expansions file in the same directory as this file?

Reply:
done